### PR TITLE
Orchestrain 2.0

### PR DESCRIPTION
**Pull request by tunahan-oguz**
New dataset classes, and models are added.
Some loss functions and metrics are debugged.
import_from keyword is removed from the configuration file necessities.
optimizer and learning scheduler were hard coded, now they can be set from the config file.
Some training fallacies are fixed.

### CODE REVIEW

**Comment by AdnanEkici** [comment]
A quick summary for changes would be perfect.
